backend/src/main/kotlin/app/ehrenamtskarte/backend/stores/geocoding/query.kt

```diff
@@ -0,0 +1,72 @@
+package app.ehrenamtskarte.backend.stores.geocoding
+
+import app.ehrenamtskarte.backend.common.COUNTRY_CODE
+import app.ehrenamtskarte.backend.common.STATE
+import app.ehrenamtskarte.backend.stores.importer.types.AcceptingStore
+import com.fasterxml.jackson.databind.ObjectMapper
+import io.ktor.client.*
+import io.ktor.client.request.*
+import io.ktor.http.*
+import org.geojson.Feature
+import org.geojson.FeatureCollection
+import java.io.File
+
+suspend fun queryFeatures(params: List<Pair<String, String>>): List<Feature> {
+    val fileName = cacheFileName(params)
+    val file = File("${System.getProperty("user.dir")}/data/nominatim/v1/$fileName.json")
```

Comment:
This path needs to be configurable in order to be able to deploy this.

Reply:
It is, for some reason you reviewed an old version. Have a look at `FeatureFetcher`

Reply:
```kotlin
val file = File("${System.getProperty("app.data")}/nominatim/v1/$fileName.json")
```

Reply:
Will remove the caching for now, so not relevant anymore.